app/templates/components/recirculation-prefooter.hbs

```diff
@@ -19,6 +16,7 @@
 						<a {{action 'adChoiceBadgeClick' bubbles=false}}
 								class="recirculation-prefooter__badge
 							recirculation-prefooter__badge-plista" href="https://www.plista.com/opt-out">
+							<img src="https://services.wikia.com/static-assets/image/c7c31771-d3e2-4aee-8d0d-6d4c4629bcff">
```

Comment:
`http://vignette.wikia.nocookie.net/c7c31771-d3e2-4aee-8d0d-6d4c4629bcff` is a proper, public-facing URL. See https://wikia-inc.atlassian.net/wiki/display/PLAT/Static+Assets